src/Node/ReadLine.purs

```diff
@@ -1,42 +1,86 @@
 -- | This module provides a binding to the Node `readline` API.
 
-module Node.ReadLine where
+module Node.ReadLine
+  ( Interface
+  , InterfaceOptions
+  , Completer
+  , LineHandler
+  , createInterface
+  , createInteractiveConsoleInterface
```

Comment:
Sorry to be a pain, but this name is a bit of a mouthful. What does the "interactive" part add? In particular, does it add anything that isn't already conveyed by "console"?